src/languageservice/jsonSchema.ts

```diff
@@ -1,3 +1,5 @@
+import { CompletionItemKind } from 'vscode-json-languageservice';
```

Comment:
Could you move import under license header

Reply:
sorry, done